app/src/assets/localization/en/protocol_command_text.json

```
@@ -34,16 +34,24 @@
   "dropping_tip_in_trash": "Dropping tip in {{trash}}",
   "engaging_magnetic_module": "Engaging Magnetic Module",
   "fixed_trash": "Fixed Trash",
+  "flex_stacker_empty": "Manually empty all labware out of Flex Stacker",
+  "flex_stacker_empty_from_location": "Manually empty all labware out of Stacker in {{slotName}}",
```

"from" is more natural? and we appear to be using "Flex Stacker" everywhere in these.
```suggestion
  "flex_stacker_empty": "Manually empty all labware from Flex Stacker",
  "flex_stacker_empty_from_location": "Manually empty all labware from Flex Stacker in {{slotName}}",
```